extensions/default/src/Components/MoreDropdownMenu.tsx

```diff
@@ -36,7 +36,7 @@ const getMenuItemsDefault = ({
       onClick: ({ commandsManager, ...props }: withAppTypes) => () => void;
     };
   }) => (
-    <DropdownMenuItem onClick={() => item.onClick({ commandsManager, ...props })}>
+    <DropdownMenuItem onClick={() => item.onClick({ commandsManager, servicesManager, ...props })}>
```

Comment:
Can you just use useSystem instead of adding more things to the dropdown menu parameter list?